### ISSUE TITLE
Beautify download page

### ISSUE THREAD

**Issue by koegel** [eclipse-theia/theia-blueprint#7]
The download page needs some love:
- A textual description what this is about: "Blueprint product to showcase Theia platform capabilities” 
- Dropdown or similar to select OS
- Some graphics for the page and icons for the download links
- Collect download stats
- Link to this project for the sources
- Link how to report bugs
- Add download page link button to Theia landing page

Currently an initial download page is available here:
https://download.eclipse.org/theia/

**Comment by koegel** [comment]
We (EclipseSource) intend to contribute to this on behalf of ST Microelectronics.

**Comment by koegel** [comment]
We are now starting to design a minimal page and are also looking into how to collect download stats.

**Comment by brianking** [comment]
If you have a draft of the content, we can pick it up and come up with some copy recommendations.

cc @croundy

**Comment by koegel** [comment]
A PR is already underway, see https://github.com/theia-ide/theia-website/pull/133.
For improvements I suggest to open a new bug and describe the improvements there or even better to open a PR once the changes of the current PR are merged.

**Comment by brianking** [comment]
Is there a timeframe for the PR to be merged? We'll wait until then before we start our review.

**Comment by koegel** [comment]
No, hopefully soon :)

**Comment by meKokabi** [comment]
Please find attached the Download page design
Design Concept: Binary Codes - Modern - Dynamic - Referencing Theia's logo visual elements
<img width="697" alt="Screen Shot 2021-01-29 at 8 22 00 AM" src="https://user-images.githubusercontent.com/75500459/106298363-ce543500-6221-11eb-822d-01dd2f443ec2.png">
<img width="838" alt="Screen Shot 2021-01-29 at 8 25 10 AM" src="https://user-images.githubusercontent.com/75500459/106298409-d7450680-6221-11eb-9e11-9d20a9d85b3e.png">

**Comment by koegel** [comment]
Thanks, but this does not match the design of the target page: https://theia-ide.org/
@jfaltermeier Can you please attach a screenshot of the current design of the download page?
IMHO: Everything is already done. Maybe you can only propose improvements and some graphics to be added. I suggest you take a look at the download page in the open PR or as shown in the screenshot Johannes will add. Thank you!

**Comment by koegel** [comment]
Also, since the PR is already underway please open a separate issue (e.g. "Improve download page design"), thanks.

**Comment by koegel** [comment]
Any further improvement efforts by @meKokabi are now pursued in #11.

**Comment by koegel** [comment]
I also opened a dedicated ticket on the website repo now: https://github.com/eclipse-theia/theia-website/issues/134

**Comment by jfaltermeier** [comment]
> Thanks, but this does not match the design of the target page: https://theia-ide.org/
> @jfaltermeier Can you please attach a screenshot of the current design of the download page?
> IMHO: Everything is already done. Maybe you can only propose improvements and some graphics to be added. I suggest you take a look at the download page in the open PR or as shown in the screenshot Johannes will add. Thank you!

The current proposal for the download page looks like this (zoomed out to fit on screen):
![ksnip_20210203-090644](https://user-images.githubusercontent.com/5889696/106716790-2793de00-65ff-11eb-9e9b-564a32bdde15.png)

**Comment by koegel** [comment]
Alpha release is planned for April 10th

**Comment by jfaltermeier** [comment]
Download page: https://theia-ide.org/docs/blueprint_download/
